README.md

```diff
@@ -15,34 +15,47 @@
 
 ## How to use
 
-1. **Install the 'JavaScript Standard Style' extension**
+1. **Install the 'StandardJS - JavaScript Standard Style' extension**
 
-   If you don't know how to install extensions in VSCode, take a look at the [documentation](https://code.visualstudio.com/docs/editor/extension-gallery#_browse-and-install-extensions).
+    Launch VSCode Quick Open (⌘+P), paste the following command, and press enter.
 
-   You will need to reload VSCode before new extensions can be used.
+    ```text
+    ext install standard.vscode-standard
+    ```
 
-2. **Install `standard`, `semistandard`, `standardx` or `ts-standard`**
+    For more information, take a look at the [documentation](https://code.visualstudio.com/docs/editor/extension-gallery#_browse-and-install-extensions).
+
+2. **Install the engine `standard`, `semistandard`, `standardx` or `ts-standard`**
 
    This can be done globally or locally. We recommend that you install them locally (i.e. saved in your project's `devDependencies`), to ensure that other developers have it installed when working on the project.
 
-3. **Disable the built-in VSCode validator**
+3. **Enable the extension**
+
+    This can be done globally or locally. We recommend that you enable it locally, to ensure that other developers have the same workflow when working on the project with VSCode.
+
+    To do this, create a new file inside your project : `.vscode/settings.json` with the following content :
 
-   To do this, set `"javascript.validate.enable": false` in your VSCode `settings.json`.
+    ```json
+    {
+      "standard.enable": true,
+      "standard.engine": "standard" /* set this to the engine you've installed previously */
+    }
+    ```
 
-## Plugin options
+## Extension options
```

Comment:
To make it consistent, we use the word `Extension` instead of `Plugin`, it means the same thing in this context.